packages/lit-dev-content/site/tutorials/content/custom-attribute-converter/01.md

```diff
@@ -1,52 +1,36 @@
+The `date` property is set in `index.html`, using JavaScript.
+The `date` property **can't** be set from an attribute, because Lit doesn't have a built-in attribute converter to convert a string date to a `Date` object.
 
-{% endaside %}
+Attribute converters tell Lit how to convert an attribute to a property and, in the case that a reactive property has the `reflect: true` flag, from property to attribute.
 
-## Attribute Converters
-
-Attribute converters tell Lit how to convert an attribute to a property and, in the case that a reactive property has the `reflect: true` flag, vice versa.
-
-### Built-in Attribute Converters
-
-Lit has several built-in attribute converters. You can choose which built-in converter is invoked by setting the `type` property of the [reactive property options](/docs/components/properties/#property-options). The built-in converters are:
+Lit has several built-in attribute converters. You can choose which built-in converter is invoked by [setting the `type` option](/docs/components/properties/#property-options) when you declare a reactive property. The built-in converters are:
 
 * `String` *(default)*
 * `Boolean`
 * `Number`
 * `Array`
 * `Object`
 
-Notice that Lit does not have a `Date` converter built-in and thus you cannot do `type: Date` to accept a date-string attribute. You can read more on how built-in coverters convert attributes to properties and vice versa in the [Lit documentation](/docs/components/properties/#conversion-type).
-
-{% aside "negative" %}
+For more information on built-in converters, see [Using the default converter](/docs/components/properties/#conversion-type).
 
-This section is meant for example does not include best-practice code.
+In this step, you'll add a new `dateStr` reactive property that accepts a human-readable date string, and can be set from an attribute. (That is, `date-str="05/05/22"`.)
 
-This section is shows why you would need custom attribute converters and is not recommended best-practice code. If you'd like to skip to the section in which you write custom attribute converters, go to [step 4](#03).
+### Add the dateStr property
 
-{% endaside %}
+* Declare a `dateStr` reactive property of type `String`.
+* Set the default value for `dateStr` to a string representation of `date`.
+* Set the attribute for `dateStr` to be `date-str`.
 
-## Goal for this step
-
-`my-element` accepts human-readable date attribute as a reactive property. e.g. `date-str="05/05/22"`
-
-### Accept a human-readable attribute string
-
-* Declare `dateStr` reactive property of type `String`
-* Set the default value to the stringified value of `date`
-* Set the attribute for `dateStr` to be `date-str`
+#### date-display.<ts-js></ts-js>
 
 {% switchable-sample %}
```

Comment:
GH won't let me do it for you – please rename `MyElement` to `DateDisplay`. I seem to have forgotten to do that in the code. I'll put out another PR for that

Reply:
Got it.